http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/SamAuthProvider.scala

```diff
@@ -181,10 +181,12 @@ class SamAuthProvider[F[_]: Effect: Logger: Timer: OpenTelemetryMetrics](samDao:
     creatorEmail: WorkbenchEmail,
     googleProject: GoogleProject
   )(implicit sr: SamResource[R], encoder: Encoder[R], ev: Ask[F, TraceId]): F[Unit] =
-    if (sr.policyNames == Set(SamPolicyName.Creator))
+    // At the time this comment was written, the else is only executed for apps
```

Comment:
Any way this comment could be made clearer for someone looking at this code in 3 months with no context? :)

Reply:
Also: technically we could start using the [createResourceWithDefaultsV2](https://sam.dsde-prod.broadinstitute.org/#/Resources/createResourceWithDefaultsV2) for runtimes and disks, which is still the `v2` API but without the inherited policies from `google-project`. Maybe as part of IA-2569 we should decide whether to just use `createResourceWithDefaultsV2` or do some larger refactoring to model runtimes/disks similar to apps.

Reply:
I updated the comment, and added notes to the ticket